### PR TITLE
My portfolio desktop version

### PR DESCRIPTION
**Pull request by Taher-web-dev**
## This branch contains the following changes: 

- Re-engineering  of html file. 
- Media query in css file to fit desktop version.

So, please take a look and give me your feedback before to merge with master branch.

### CODE REVIEW

File: styles/template2.css [hunk 1]
```diff
@@ -1,3 +1,158 @@
...
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 600px;
+}
+
+@media screen and (min-width: 750px) {
```
Comment (safafa) [inline styles/template2.css:92]: - [ ] For the Desktop breakpoint the value for `min-width` property should be in range of `992px - 1100px` your break point is 750px kindly fix the issue.

image
original design :
image

- [ ] As you can see from the pictures for the desktop view the headline section looks a bit different from the one in the original design. The background image doesn't fit the whole screen. Kindly fix the issue.

image
original design:
image

- [ ] The project's cards also look a bit different from the original design. I think you need to adjust the margins and center the cards. Kindly fix the issue.

original design : 
image
your webpage:
image

- [ ] The about me section also looks a bit different. It has 2 blocks The left one takes 40% and the right one the remaining 60% of the page's width. The elements of each bloc are aligned to the right. The languages cards and the borders for the technologies look also different Kindly fix the issue.

image
image
mobile view:
image

original design:
image
mobile view:
image

- [ ] Kindly fix the background image of the contact-me section for desktop and mobile view.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @taherhagui  , You did a great job so far 👏🏼 👏🏼 

- The linter checks are passing.
- The repository is professional.
- the Readme is descriptive.

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Using one media query instead of many media queries for the same breakpoint. 

 Usind ong

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 2]
```diff
@@ -40,53 +45,50 @@ <h3>LET'S CONNECT</h3>
                 </div>
             </footer>
         </section>
+      </section>
       <section id="my_work">
         <section id="p1">
```
Comment (Alexoid1) [inline index.html:50]: - [ ] Kindly add a background color for the body `#E5E5E5` and for the sections set a #FFF background. This is what makes look your project different from the original design.
Captura de pantalla_2021-09-24_21-11-08

**Figma**
Captura de pantalla_2021-09-24_21-05-09

File: index.html [hunk 3]
```diff
@@ -3,25 +3,30 @@
...
             <nav><img src="statics/images/Union.png" alt="Menu"></nav>
+            <ul class="internal-link">
+               <li><a href="#my_work">Portfolio</a></li>
+               <li><a href="#about_me">About</a></li>
+               <li><a href="#form-section"> Contact </a></li>
+            </ul>
 
           </nav>
         </header>
         <section id="home_presentation">
```
Comment (Alexoid1) [inline index.html:27]: - [OPTIONAL] Kindly style the bottom of the `home_presentation` section as the original design. The `border-bottom has a rounded corner.
Captura de pantalla_2021-09-24_21-18-57

File: index.html [hunk 4]
```diff
@@ -40,53 +45,50 @@ <h3>LET'S CONNECT</h3>
...
+                      A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <footer>
+                  <ul id="skills_0" class="skills">
+                     <li>html</li>
+                     <li>css</li>
+                     <li>javaScript</li>
+                  </ul>
+                  <div class="external_link">
+                     <a href="#">See Project</a>
```
Comment (Alexoid1) [inline index.html:71]: - [OPTIONAL] For each `See Project` button the text color change to white on hover.

File: index.html [hunk 5]
```diff
@@ -176,40 +171,43 @@ <h2>LET’S CONNECT</h2>
...
+                     </li>
+                     <li class="pro">
+                       <img src="statics/images/html.png" alt="html image">
+                       <p>HTML</p>
+                     </li>
+                     <li class="pro">
+                       <img src="statics/images/css.png" alt="css image">
+                       <p>CSS</p>
+                     </li>
+                </ul>
```
Comment (Alexoid1) [inline index.html:194]: - [ ] The languages section looks quite different than the original design the alignment change to row and the containers turn to squares with the logo at the top-left corner.
Captura de pantalla_2021-09-24_21-47-40

**Figma**
Captura de pantalla_2021-09-24_21-47-23

Comment (Alexoid1) [reply index.html:194]: - [OPTIONAL] The container also has a rounded corner.
Captura de pantalla_2021-09-24_21-49-36

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @Taher-web-dev,

Impressive project!
You are doing a great job so far, and the project is looking good.

## Required Changes :recycle:
For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Taher-web-dev , 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**